src/routes/github/create-branch/github-create-branch-options-get.test.ts

```diff
@@ -3,6 +3,16 @@ import supertest from "supertest";
 import { getLogger } from "config/logger";
 import { getFrontendApp } from "~/src/app";
 import { getSignedCookieHeader } from "test/utils/cookies";
+import { booleanFlag, BooleanFlags } from "config/feature-flags";
+import { when } from "jest-when";
+
+jest.mock("config/feature-flags");
+
+const turnFF_OnOff = (newStatus: boolean) => {
```

Comment:
/nit
```suggestion
const turnFF_On = (newStatus: boolean) => {
```

Reply:
The function's job isn't to only turn the ff on otherwise we could just hardcode the value to `true`. The idea is that if you have multiple tests, you call that function and can turn it on/off based on what you pass in.

Reply:
IMO, if we name the variable to `turnFF_On` and if we pass `true`, that means its **ON** and if we pass `false`, it means its **OFF**.
But when its `turnFF_OnOff` and we pass `true`, its not clear as if the **ON** part is true or the **OFF** part is true. I had to check the code and deduce when to pass true/false values. 
But yes, its def not a blocker, so feel free to leave it as it.